cpu/stm32/include/periph/cpu_common.h

```diff
@@ -113,6 +113,12 @@ typedef enum {
 #endif
 } bus_t;
 
+typedef struct periph_t
+{
+    volatile uint32_t *en_reg;
+    uint32_t en_mask;
+} periph_t;
```

Comment:
thoughts on the name of this `sturct`?

Reply:
the name seems a bit generic, at least the usage in this PR seems like `rcc_register_t` or more specific